tests/runtime/graph_ops/test_embed_model_property.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import onnx
+from functools import partial
+from onnx import helper
+from onnx import AttributeProto, TensorProto, GraphProto
+from vortex.runtime.onnx.graph_ops.helper import get_metadata_prop
+from vortex.runtime.onnx.graph_ops.check_metadata import check_metadata
+
+from .dummy_model import dummy_model
+
+# shorter version of registry
+from vortex.development import ONNX_GRAPH_OPS as graph_ops
+
+out_fmt = dict(
+    class_label=dict(
+        indices=[0],
+        axis=0,
+    ),
+    class_confidence=dict(
+        indices=[1],
+        axis=0,
+    )
+)
+
+class_names = ['cat', 'dog']
+
+def test_embed_model_property():
+    model = dummy_model()
+    props = dict(
+        output_format=out_fmt,
+        class_names=class_names,
+        other_prop=1234
+    )
+    op = graph_ops.create_from_args('EmbedModelProperty', props=props)
```

Comment:
the `EmbedModelProperty.parse` is not yet tested

Reply:
well it is actually only re-use other ops
```
def parse(cls, model: onnx.ModelProto) -> Dict[str,Any]:
      output_format = EmbedOutputFormatMetadata.parse(model)
      class_names = EmbedClassNamesMetadata.parse(model)
      return dict(output_format=output_format,class_names=class_names)
```
which is tested below

but yeah it'd be nice we also test that, please wait

Reply:
yepp, there might be other thing inside that would be added as well just in case